### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by HaleySchuhl**
**Describe your changes**
CMYK and other doc pages minor updates 

**Type of update**
Is this a:
* Update to documentation

**Associated issues**
Reference associated issue numbers. Does this pull request close any issues?

**Additional context**

**For the reviewer**
See [this page](https://plantcv.readthedocs.io/en/stable/pr_review_process/) for instructions on how to review the pull request.

- [x] Test coverage remains 100%
- [x] Documentation tested
- [x] Code reviewed
- [x] PR approved

### CODE REVIEW

File: docs/transform_resize.md [hunk 1]
```diff
@@ -69,19 +69,23 @@ resize_img5 = pcv.transform.resize_factor(img=img, factors=(0.1154905775, 0.1154
...
+
 ![Screenshot](img/documentation_images/resize/19_resize2.jpg)
 
-3rd resized image (cropping)
+**3rd resized image (cropping)**
+
 ![Screenshot](img/documentation_images/resize/19_resize3.jpg)
 
-3rd resized image (zero-padding)
+**3rd resized image (zero-padding)**
```
Comment (jdavidpeery) [inline docs/transform_resize.md:83]:
```suggestion
**4th resized image (zero-padding)**
```

File: docs/transform_resize.md [hunk 2]
```diff
@@ -69,19 +69,23 @@ resize_img5 = pcv.transform.resize_factor(img=img, factors=(0.1154905775, 0.1154
...
+
 ![Screenshot](img/documentation_images/resize/19_resize4.jpg)
 
-5th resize image
+**Resize by factor image**
```
Comment (jdavidpeery) [inline docs/transform_resize.md:87]:
```suggestion
**5th resized image (by factor)**
```

File: docs/transform_resize.md [hunk 3]
```diff
@@ -69,19 +69,23 @@ resize_img5 = pcv.transform.resize_factor(img=img, factors=(0.1154905775, 0.1154
 ```
 
 **Images after resizing**
-1st resized image
+
```
Comment (jdavidpeery) [inline docs/transform_resize.md:72]:
```suggestion
**1st resized image (default interpolation)

```

File: docs/transform_resize.md [hunk 4]
```diff
@@ -69,19 +69,23 @@ resize_img5 = pcv.transform.resize_factor(img=img, factors=(0.1154905775, 0.1154
...
 ![Screenshot](img/documentation_images/resize/19_resize1.jpg)
 
-2nd resized image
+**2nd resized image**
```
Comment (jdavidpeery) [inline docs/transform_resize.md:75]:
```suggestion
**2nd resized image (Interpolation by nearest)**
```

**Review by jdavidpeery — COMMENTED** [review]
Looks good! Just suggested a few changes on one of them

**Review by jdavidpeery — APPROVED** [review]
Looks good

File: docs/parallel_config.md [hunk 5]
```diff
@@ -207,3 +207,6 @@ Windows you will need to specify the whole path to `plantcv-workflow.py`.
 ```shell
 python %CONDA_PREFIX%/Scripts/plantcv-workflow.py --config my_config.json
 ```
+
+**Source Code:** [Here](https://github.com/danforthcenter/plantcv/blob/811e6558a5b4fbf7e5781a9d92834a0b19642bb2/plantcv/parallel/__init__.py)
```
Comment (nfahlgren) [inline docs/parallel_config.md:211]: This URL looks like it's to a commit rather than a branch, I assume that's not what was intended?

Comment (HaleySchuhl) [reply docs/parallel_config.md:211]: Oh true. I can fix that!

**Pull request merged**